src/Uno.UWP/ApplicationModel/Calls/PhoneCallManager.cs

```diff
@@ -6,17 +6,13 @@
 
 namespace Windows.ApplicationModel.Calls
 {
-	public partial class PhoneCallManager
+	public static partial class PhoneCallManager
```

Comment:
I think we should consider those changes as breaking.

Reply:
Yeah I forgot to add `!` to the first commit message :smile:

Will do that and force-push. Anything else other than the commit message? doc somewhere, etc?